nav2_smac_planner/lattice_primitives/helper.py

```diff
@@ -17,7 +17,7 @@
 
 def normalize_angle(angle):
     """
-    Normalize the angle to between [-pi, pi).
+    Normalize the angle to between (-pi, pi].
```

Comment:
I'll need to test this locally too with a few of these files. I thought we needed 0-2PI not -PI to PI. I don't see how it was working before looking at the parsers https://github.com/ros-planning/navigation2/blob/main/nav2_smac_planner/src/node_lattice.cpp but clearly it was so I need to understand what's going on there